### PR TITLE
Adds Table of content to Readme.md

### PR DESCRIPTION
**Pull request by shivabhusal**
Though there is a link to friendly navigable format to this doc, I think it's hard to find/notice. Also, I think it would be much helpful to have TOC on the readme, as it is getting hefty.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,50 @@ It's composed of 10 modules:
 * [Validatable](http://rubydoc.info/github/plataformatec/devise/master/Devise/Models/Validatable): provides validations of email and password. It's optional and can be customized, so you're able to define your own validations.
 * [Lockable](http://rubydoc.info/github/plataformatec/devise/master/Devise/Models/Lockable): locks an account after a specified number of failed sign-in attempts. Can unlock via email or after a specified time period.
 
+## Table of Content
```
Comment (tegon) [inline README.md]: Can you change here to **Table of Contents**?

**Comment by tegon** [comment]
@shivabhusal Thank you for your contribution! ❤️ 
This can indeed be helpful, I just comment with a little suggestion, I hope you don't mind 😄

File: README.md (hunk 1 above)
Comment (shivabhusal) [reply README.md]: sure

Comment (shivabhusal) [reply README.md]: Done!

**Pull request merged**